burst/providers/burst_overrides.py

```diff
@@ -122,7 +122,7 @@ def source():
     # Rutor
     'rutor': {
         'anime_query': "0/10/300/2/QUERYEXTRA",
-        'movie_query': "0/1/300/2/QUERYEXTRA",
+        'movie_query': "0/0/300/2/QUERYEXTRA",
```

Comment:
I don't like that way (making movie search match anything, not even video, can be music as well). 
Will accept for testing how it feels.

Reply:
This site can't to take multiple category in search request. Only one.